### DISCUSSION TITLE
emqx plugins start failure

### DISCUSSION THREAD

**Discussion by xuebear** [emqx/emqx#6321] [Q&A] [answered]
I tried to start the Redis plugin，Get the following information：
`bash-5.0$ ./emqx_ctl plugins load emqx_auth_redis
Load plugin emqx_auth_redis error: {emqx_auth_redis,
                                    {bad_return,
                                     {{emqx_auth_redis_app,start,[normal,[]]},
                                      {'EXIT',
                                       {{badmatch,
                                         {error,
                                          {shutdown,
                                           {failed_to_start_child,
                                            emqx_auth_redis,
                                            {shutdown,
                                             {failed_to_start_child,
                                              worker_sup,
                                              {shutdown,
                                               {failed_to_start_child,
                                                {worker,1},
                                                {connection_error,
                                                 econnrefused}}}}}}}}},
                                        [{emqx_auth_redis_app,start,2,
                                          [{file,"emqx_auth_redis_app.erl"},
                                           {line,30}]},
                                         {application_master,start_it_old,4,
                                          [{file,"application_master.erl"},
                                           {line,277}]}]}}}}}.`

help me!

**Comment by zmstone** [comment, marked as answer]
econnrefused hints the remote end refused the connection request

**Reply by xuebear** [reply]
I solved the problem according to the ideas you provided. Thank you and love you.
